src/bundle.c

```diff
@@ -1389,6 +1346,12 @@ static gboolean open_local_bundle(RaucBundle *bundle, GError **error)
 		goto out;
 	}
 
+	res = data_is_parsable_cms(bundle->sigdata);
```

Comment:
This is a duplicate of the call at the end of `open_remote_bundle()`, so we should move it out of the local/remote if/else `check_bundle()`. At that point it would be directly before the call to `cms_is_detached()` which also has to parse the CMS. So we could simplify this by adding better error reporting for invalid CMS data in `cms_is_detached()`.